app/javascript/mastodon/features/ui/components/column_loading.tsx

```diff
@@ -0,0 +1,9 @@
+import Column from 'mastodon/components/column';
+import ColumnHeader from 'mastodon/components/column_header';
+
+export const ColumnLoading: React.FC = () => (
+  <Column>
+    <ColumnHeader />
+    <div className='scrollable' />
+  </Column>
+);
```

Comment:
This incorrectly drops the `multiColumn` prop, sometimes causing an infinite loop in advanced mode.

Reply:
I did it because the type check was throwing errors about it. Hm.